### ISSUE TITLE
[false positives] SCSS variables are tracked as packages, while they are not

### ISSUE THREAD

**Issue by theKashey** [depcheck/depcheck#633]
### Bug Description

In some circumstances _@including_ a mixin is treated as an import

### Code snippets

Where the dependency is used:

```scss
@use "~@ui/styles/theme";
@use "~@ui/styles/mixin";

.icon {
  span {
    @include mixin.visuallyHidden;
    // mixin is missing in project dependencies
  }
}
```

### Extra info

The [current code](https://github.com/depcheck/depcheck/blob/main/src/parser/sass.js#L58) basically reacts at any "@at" rules.
I've tried to filter only `import`/`use` and it seems to work fine.

I can draft a PR if such solution is acceptable.

**Comment by theKashey** [comment]
Technically speaking the required amount of parsing can be achieved via simple regexp - there is no real need to build full ast, solving #632 as well.

**Comment by theKashey** [comment]
Still valid issue

**Comment by ptmkenny** [comment]
I'm only seeing a potentially related issue with media queries.

**styles.scss**

```
@media only screen
  and (min-device-width: 375px) {
  .cropContainer {
    height: 250px;
  }
}
```

And `depcheck` reports:

`Missing dependencies
* only: ./styles.scss`
